super.c

```diff
@@ -159,6 +173,9 @@ static int simplefs_sync_fs(struct super_block *sb, int wait)
 
 static int simplefs_statfs(struct dentry *dentry, struct kstatfs *stat)
 {
+#ifdef DEBUG
```

Comment:
Don't do this. Minimize the necessary changes for Linux 6.x compatibility.

Reply:
Sure